blockchain/chain/src/store/base_fee.rs

```diff
@@ -90,7 +90,7 @@ where
     // Compute next base fee based on the current gas limit and parent base fee.
     let parent_base_fee = ts.blocks()[0].parent_base_fee();
     Ok(compute_next_base_fee(
-        parent_base_fee,
+        &parent_base_fee.clone(),
```

Comment:
I think we don't need `&` `.clone()`, just parent_base_fee

Reply:
Good catch! Done.